/***************************************************************************
 *   Copyright (C) 2011 by Francesco Cecconi                               *
 *   francesco.cecconi@gmail.com                                           *
 *                                                                         *
 *   This program is free software; you can redistribute it and/or modify  *
 *   it under the terms of the GNU General Public License as published by  *
 *   the Free Software Foundation; either version 2 of the License.        *
 *                                                                         *
 *   This program is distributed in the hope that it will be useful,       *
 *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
 *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
 *   GNU General Public License for more details.                          *
 *                                                                         *
 *   You should have received a copy of the GNU General Public License     *
 *   along with this program; if not, write to the                         *
 *   Free Software Foundation, Inc.,                                       *
 *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
 ***************************************************************************/

#ifndef ADDVULNERABILITYURL_H
#define ADDVULNERABILITYURL_H

#include <QtGui/QDialog>
#include <QtCore/QObject>

#include "ui_addvulnerabilityurl.h"

class addVulnerabilityUrl : public QDialog, private Ui::addUrl
{
    Q_OBJECT

public:
    addVulnerabilityUrl(QObject* parent=0);
    ~addVulnerabilityUrl();

private slots:
    void exit();

signals:
    void doneUrl(const QString urlName, const QString urlAddr);
};

#endif
